lib/scss/stanford_events.node.scss

```diff
@@ -12,6 +12,16 @@
     @include modular-typography(5);
   }
 
+  .su-event-label-past {
+    color: #7f7776; //$su-color-accent-stone
```

Comment:
Why not use the variable here?

Reply:
It isn't on the new Decanter with it yet. That is in the next sprint. I thought the comment would work as a reminder to update.

Reply:
Ah, perhaps add to the comment with that detail. Thank you.

Reply:
I fixed the comment. :)